FlowCrypt/Controllers/Inbox/InboxViewController+TableView.swift

```diff
@@ -341,55 +360,103 @@ extension InboxViewController {
         return inboxInput[safe: index]
     }
 
-    func perform(action: MessageAction, at indexPath: IndexPath) {
-        guard let inboxItem = inboxItem(at: indexPath) else { return }
+    private func performPermanentDeleteAction(on inboxItems: [InboxItem]) {
+        showPermanentDeleteThreadAlert(onAction: { [weak self] _ in
```

Comment:
This alert shows `You're about to permanently delete a message` text, let's change it to be dynamic and say `You're about to permanently delete X threads`

<img width="200" alt="Screenshot 2023-08-21 at 21 22 32" src="https://github.com/FlowCrypt/flowcrypt-ios/assets/1062093/0991b817-f303-4025-8bc9-e4299484c7e6">